src/main/java/dk/kb/image/IIPFacade.java

```diff
@@ -0,0 +1,241 @@
+        // http://example.com/iipsrv.fcgi?FIF=/mymount/85/c1/85c1df89-bffe-48e0-8813-111f6f0fba50.jp2&CVT=jpeg
+        UriBuilder builder = UriBuilder.
+                fromUri(ServiceConfig.getConfig().getString(KEY_IIP_SERVER)).
+                queryParam("FIF", FIF); // Mandatory
+        ProxyHelper.addIfPresent(builder, "WID", WID);
+        ProxyHelper.addIfPresent(builder, "HEI", HEI);
+        ProxyHelper.addIfPresent(builder, "RGN", RGN);
+        ProxyHelper.addIfPresent(builder, "QLT", QLT);
+        ProxyHelper.addIfPresent(builder, "CNT", CNT);
+        ProxyHelper.addIfPresent(builder, "SHD", SHD);
+
+        ProxyHelper.addIfPresent(builder, "LYR", LYR);
+        ProxyHelper.addIfPresent(builder, "ROT", ROT);
+        ProxyHelper.addIfPresent(builder, "GAM", GAM);
+        ProxyHelper.addIfPresent(builder, "CMP", CMP);
+        ProxyHelper.addIfPresent(builder, "PFL", PFL);
+        ProxyHelper.addIfPresent(builder, "CTW", CTW);
+        ProxyHelper.addIfPresent(builder, "INV", INV); // TODO: Figure out how to add INV as a value-less param
+        ProxyHelper.addIfPresent(builder, "COL", COL);
+
+        ProxyHelper.addIfPresent(builder, "JTL", JTL);
+        ProxyHelper.addIfPresent(builder, "PTL", PTL);
+        ProxyHelper.addIfPresent(builder, "CVT", CVT);
+
+        final URI uri = builder.build();
+
+        return ProxyHelper.proxy(FIF, uri, requestURI);
+    }
+
+    public javax.ws.rs.core.StreamingOutput getDeepzoomDZI(
+            URI requestURI,
+            String imageid) throws ServiceException {
+        validateDeepzoomDZIRequest(imageid);
+
+        // Defaults
+
+        // TODO: Use the UriTemplate system like IIIFFacade
+        // http://example.com//fcgi-bin/iipsrv.fcgi?DeepZoom=/your/image/path.tif.dzi
+        UriBuilder builder = UriBuilder.
+                fromUri(ServiceConfig.getConfig().getString(KEY_DEEPZOOM_SERVER)).
+                path(imageid + ".dzi"); // Mandatory
+
+        final URI uri = builder.build();
+
+        return ProxyHelper.proxy(imageid, uri, requestURI);
+    }
+
+    public javax.ws.rs.core.StreamingOutput getDeepzoomTile(
+            URI requestURI,
+            String imageid, Integer layer, String tiles, String format, Float CNT, List<Integer> SHD,
+            Float GAM, String CMP, String CTW, Boolean INV, String COL) throws ServiceException {
+        validateDeepzoomTileRequest(imageid, layer, tiles, format, CNT, SHD, GAM, CMP, CTW, INV, COL);
+
+        // Defaults
+        if (format == null || "jpg".equals(format)) {
+            format = "jpeg";
+        }
+
+        // TODO: Use the UriTemplate system like IIIFFacade
+        // http://example.com//fcgi-bin/iipsrv.fcgi?DeepZoom=/your/image/path.tif.dzi
+        UriBuilder builder = UriBuilder.
+                fromUri(ServiceConfig.getConfig().getString(KEY_DEEPZOOM_SERVER)).
+                path(imageid + "_files").path("" + layer).path(tiles + "." + format); // Mandatory
```

Comment:
Sudden realization: Deepzoom can be accessed both as path and as query params. We need to handle this in the configuration & code. We need to discuss how to do so.